webapp/platform/client/src/websocket.ts

```diff
@@ -28,7 +43,7 @@ export default class WebSocketClient {
     // server-sent event stream.
     private serverSequence: number;
     private connectFailCount: number;
-    private responseCallbacks: {[x: number]: ((msg: any) => void)};
+    private responseCallbacks: { [x: number]: ((msg: any) => void) };
```

Comment:
Unneeded change? (wonder why is it not caught by the linter)

Reply:
Dropped this change.